### PR TITLE
#160053423 Setup an empty react project

### PR DESCRIPTION
**Pull request by AGMETEOR**
#### What does this PR do?
This PR sets up a react project using create-react-app. 
#### Description of Task
Have a working react project set up and pushed to Github
#### How should this be manually tested?
cd  into ah-frontend-taces and run $npm install and then $npm start
#### Background
The project has been set up to use CSS modules. Instead of CSS files being imported like so "import './App.css'" they will instead be imported as classes like so "import classes from './App.css'". 
#### What are the relevant pivotal tracker stories?
[#160053423](https://www.pivotaltracker.com/story/show/159951810)

### CODE REVIEW

**Comment by abulojoshua1** [comment]
Thanks, @AGMETEOR for this good work. But, do we need the image file at [src/logo.svg](https://github.com/andela/ah-frontend-taces/blob/ch-set-up-react-160053423/src/logo.svg)?

**Comment by lytes20** [comment]
Hey @AGMETEOR  , you have a lot of files. Could you please clarify on what the `config` and `scripts` folders are for?

**Comment by lytes20** [comment]
Do we need all those files in the `src` folder, shouldn't it be empty?

**Review by ssewilliam — COMMENTED** [review]
I believe most of this work was generated automatically as @AGMETEOR, Initiated that he will be using the automatic way of setting up the React project. Folder contents represent the least required functionality.
With that, I believe this is good.

**Review by ssewilliam — APPROVED** [review]

**Review by abulojoshua1 — APPROVED** [review]

**Comment by AGMETEOR** [comment]
Gideon, a lot of the files are auto-generated by the create-react-app. However, in the src folder, there's a basic component set up. I left those files there because I wanted to show how the CSS using CSS modules would be imported. Basically, CSS modules allow us to import CSS as if it was a JSON object, hence the "classes.<css file>" import style you are seeing.

**Comment by geofrocker** [comment]
@AGMETEOR Remove all the unnecessary files from `create-react-app`. We do not forexample need the logo, serviceworker.js and many more files

**Comment by AGMETEOR** [comment]
@geofrocker I think the serviceworker.js file is important for caching purposes.
https://stackoverflow.com/questions/47953732/what-does-registerserviceworker-do-in-react-js

**Comment by geofrocker** [comment]
@AGMETEOR we are not using it in this project

**Comment by AGMETEOR** [comment]
@geofrocker I have removed it. I npm run ejected and some more folders were generated that are now important to running this application. Any other particular files I should take out?

**Review by RutaleIvanPaul — APPROVED** [review]

**Review by phillipseryazi — APPROVED** [review]
this PR is good to go

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import App from './App';
+
+ReactDOM.render(<App />, document.getElementById('root'));
+registerServiceWorker();
```
Comment (geofrocker) [inline src/index.js]: Are you still using this? Take your time and remove all the unnecessary lines and files @AGMETEOR

**Review by lytes20 — APPROVED** [review]

File: src/index.js (hunk 1 above)
Comment (AGMETEOR) [reply src/index.js]: Oopps! my bad. I have removed that.

**Pull request merged**